app/worker/PDFWorker.js

```diff
@@ -16,6 +16,7 @@
  */
 
 /* global importScripts, postMessage, __DEV__ */
+import i18next from '../core/language';
```

Comment:
This one is relevant, because it's in a web worker separate from the main context, and language needs to be loaded there for viz. But I can indeed just do `import '../core/language';`